src/api/app/views/webui2/webui/package/_bottom_actions.html.haml

```diff
@@ -0,0 +1,56 @@
+%ul.list-inline.mb-0
```

Comment:
This view has a lot of `if/else` and it's difficult to follow what is going on.

Reply:
I know :disappointed:, but I only move the code into a partial